src/mobu/main.py

```diff
@@ -38,6 +41,14 @@
 __all__ = ["create_app", "lifespan"]
 
 
+if os.environ.get("MOBU_SENTRY_DSN"):
+    sentry_init(
+        dsn=config_dependency.config.sentry_dsn,
+        env=config_dependency.config.sentry_environment,
+        traces_sample_config=config_dependency.config.sentry_traces_sample_config,
+    )
```

Comment:
This is mostly a style thing, but I'd put this inside `create_app` so that we're not running some code on module import and other code on uvicorn startup unless there's some reason why they really need to happen at different times.

Reply:
That should be fine, the only reason we'd want it at module level is if we wanted to catch exceptions in some other module-level logic happening in the module, of which there is none here.